comparisons/validate.C

```diff
@@ -60,6 +61,19 @@ bool stepContainsNU(const TString& s, TString v){
   }
 }
 
+std::vector<TString>  getBranchNames(const TString& bPattern){
+  TObjArray * all_branches = Events->GetListOfBranches();
```

Comment:
@slava77 I can augment the list of branches with the one matching in the refEvents

Reply:
IIRC, ref is loaded and plotted first. So, using `refEvents` would be better.